### ISSUE TITLE
Disabled Social Tab still shows up for post type - separately

### ISSUE THREAD

**Issue by michaeltorbert** [semperfiwebdesign/all-in-one-seo-pack#1077]
<img width="321" alt="screen shot 2017-08-10 at 3 56 28 pm" src="https://user-images.githubusercontent.com/17364998/29189493-c4c49f7c-7de4-11e7-99cf-410bc9e3aadf.png">

<img width="975" alt="screen shot 2017-08-10 at 3 58 09 pm" src="https://user-images.githubusercontent.com/17364998/29189492-c4b6d1a8-7de4-11e7-9fbd-a581e3961e66.png">

**Comment by arnaudbroes** [comment]
@michaeltorbert 
I noticed this issue with Steve earlier as well, but I'm having a hard time reproducing it. 
It seems to be appearing at random. I'm testing this with fresh installs each time, and when I use the same plugin version, it either appears or it doesn't.

**Comment by michaeltorbert** [comment]
@arnaudbroes It's definitely not random. :)
When you're able to reproduce it, export the settings. That way you can import them into another site if needed.

**Comment by wpsmort** [comment]
@arnaudbroes I can now reliably reproduce this bug.  Here are the steps to follow:

1. On a new install of WordPress, install and activate AIOSEOP from the WP.org plugin repository
2. Activate the Social Meta module in Feature Manager
3. Edit a post and the problem should be visible

I can also reproduce this by resetting all settings in AIOSEOP to default and then activating the Social Meta module.

Can you confirm you can reproduce this and then track this down to the version and commit where this bug originated.

**Comment by arnaudbroes** [comment]
@wpsmort 
Thanks, that worked! Traced back to 2.3.14.
One of these commits (same as #1053 #1074 ) caused the issue:

56db40c
04184bb
eaf66d8
c50b65d
762f9e3

**Comment by michaeltorbert** [comment]
@wpsmort @arnaudbroes Can you determine exactly what the setting is that's breaking this? Maybe comparing two settings dumps... one that isn't broken with one that's broken but shouldn't be?

**Comment by wpsmort** [comment]
@michaeltorbert The setting that breaks this is the Enable Facebook Meta for Post Types setting.  This is caused because the checkboxes for Posts and Pages were unchecked by default in v2.3.14. 

We've now fixed that bug (Issue #1053) in the 2.3.16 branch and I've confirmed that it also fixes this issue as well.  I tested this by installing v2.3.14, confirmed the bug is present and then updated to 2.3.16 and confirmed the bug is no longer present.
